charts/tempo-distributed/README.md

```diff
@@ -19,6 +19,13 @@ helm repo add grafana https://grafana.github.io/helm-charts
 ## Upgrading
 
 A major chart version change indicates that there is an incompatible breaking change needing manual actions.
+### From Chart version < 0.21.0
```

Comment:
This isn't a breaking change so we don't need an upgrade guide. We should also only increment the patch version instead of the minor version: `0.20.0` -> `0.20.1`

Reply:
Thank you @joe-elliott for reviewing my PR. I've updated the chart version.